password_security/migrations/17.0.2.0.0/pre-migration.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2024 Akretion France (http://www.akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.rename_fields(
```

Comment:
Why are company fields being renamed if `ir.config_parameter` is replacing them? Shouldn't pre-migration copy the fields into `ir.config_parameter`, then post-migration drop the fields?